platform/darwin/src/MGLStyleValue.h

```diff
@@ -6,20 +6,70 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-// TODO: API docs
+/**
+ Options for `MGLStyleValue` function objects.
+ */
 typedef NSString *MGLStyleFunctionOption NS_STRING_ENUM;
 
-// TODO: API docs
+/**
+ An `NSNumber` object containing an integer.
```

Comment:
The first paragraph of the comment is the abstract; the abstract should be unique within the header or class. In this case, add "that determines the style function's exponential interpolation base".